src/test/java/carracing/test/PositionRestrictionsStrategy.java

```diff
@@ -0,0 +1,19 @@
+package carracing.test;
```

Comment:
일반적으로 테스트 소스의 패키지는 서비스 코드의 패키지를 따라가요.
```suggestion
package carracing.domain.car;
```